src/pages/Search.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
-import Header from '../components/Header';
-
-const Search = () => (
-  <>
-    <Header />
-    <div>Search placeholder</div>
-  </>
-);
+import { useParams } from 'react-router-dom';
+
+
+const Search = () => {
+  const { subreddit } = useParams();
```

Comment:
This implementation is totally fine. Just one advice to make it a bit cleaner imo: instead of setting the default below you can do it here

```
const { subreddit = 'javascript' } = useParams();
```